WordPress/WordPressTest/NotificationSyncMediatorTests.swift

```diff
@@ -86,7 +81,7 @@ class NotificationSyncMediatorTests: XCTestCase {
 //        OHHTTPStubs.stubRequest(forEndpoint: endpoint, withFileAtPath: stubPath)
 //
 //        // Make sure the collection is empty, to begin with
-//        XCTAssert(manager.mainContext.countObjects(ofType: Notification.self) == 0)
+//        XCTAssert(mainContext.countObjects(ofType: Notification.self) == 0)
```

Comment:
I wonder if this commented test could be removed instead...

🤔 Let's just say it's out of scope for this PR and that it's okay to change the code so, if we'll uncomment it, it's more likely it'll work out of the box.